README.md

```diff
@@ -31,6 +33,7 @@ With Applications in R](http://fs2.american.edu/alberto/www/analytics/ISLRLectur
 - [Deep reinforcement learning Course](https://github.com/simoninithomas/Deep_reinforcement_learning_Course)
 - [Simple Reinforcement Learning with Tensorflow](https://medium.com/emergent-future/simple-reinforcement-learning-with-tensorflow-part-0-q-learning-with-tables-and-neural-networks-d195264329d0)
 - [Reinforcement Learning by Georgia Tech](https://www.udacity.com/course/reinforcement-learning--ud600)
+- [Advanced R](http://adv-r.had.co.nz/)
 
```

Comment:
Hmmm, I think we should have a `### Language` section for things like this.

Reply:
Ok